### DISCUSSION TITLE
Screenshots with `vtkLocalView`

### DISCUSSION THREAD

**Discussion by banesullivan** [Kitware/trame#181] [Q&A]
`examples/validation/core/24_vtk_download_image.py` provides an example of how to download a screenshot with `VtkRemoteView`, but how would I take a screenshot with `vtkLocalView`?

**Comment by jourdain** [comment]
That would require JS calls that are not exposed by default but available on the vtk.js API layer.

**Comment by jourdain** [comment]
I guess we should list all the feature and how we want to expose them (API) as it is straight forward to add such thing...
